clouddriver-kubernetes/src/main/groovy/com/netflix/spinnaker/clouddriver/kubernetes/v1/provider/view/KubernetesV1InstanceProvider.groovy

```diff
@@ -68,6 +78,38 @@ class KubernetesV1InstanceProvider implements InstanceProvider<KubernetesV1Insta
 
   @Override
   String getConsoleOutput(String account, String region, String id) {
-    return null
+    KubernetesNamedAccountCredentials<KubernetesV1Credentials> credentials;
+    try {
+      credentials = (KubernetesNamedAccountCredentials) accountCredentialsProvider.getCredentials(account)
+    } catch(Exception e) {
+      log.warn("Failure getting account credentials for ${account}")
+      return null
+    }
+    if (credentials == null || credentials.getProviderVersion() != ProviderVersion.v1) {
```

Comment:
You can abbreviate to `if (credentials?.getProviderVersion() != ProviderVersion.v1) {`

Reply:
Sweet, will do!